.github/workflows/build-and-deploy.yml

```diff
@@ -250,9 +250,20 @@ jobs:
           dir="$(cygpath -au artifacts)" &&
           mkdir -p "$dir" &&
 
+          case "$ARCHITECTURE" in
+            aarch64)
+              GIT_EXE="/clangarm64/bin/git.exe"
+              BUILD_SRC=
+              ;;
+            *)
+              GIT_EXE="/mingw64/bin/git.exe"
+              BUILD_SRC="YesPlease"
+              ;;
+          esac &&
+
           {
             test -f /usr/bin/git ||
-            printf '#!/bin/sh\n\nexec /mingw64/bin/git.exe "$@"\n' >/usr/bin/git
+            printf '#!/bin/sh\n\nexec '$GIT_EXE' "$@"\n' >/usr/bin/git
```

Comment:
I think we could theoretically use `$MINGW_PREFIX/bin/git.exe` instead of setting `$GIT_EXE` in that `case`.